src/Collection/CollectionTrait.php

```diff
@@ -140,6 +140,74 @@ public function some(callable $callback): bool
         return false;
     }
 
+    /**
+     * @inheritDoc
+     */
+    public function none(callable $callback): bool
```

Comment:
Couldn't this method be a negation of `some()`?

Reply:
yes, it could be.